Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:current
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY package.json .
+RUN npm install -g npm@latest
+RUN npm install
+RUN npm install -g @angular/cli
```

Comment:
Not needed, `npx` command will use the angular cli installed by `npm install` .

Reply:
Ok